ops/model.py

```diff
@@ -3250,7 +3256,7 @@ def relation_get(
             raise TypeError('is_app parameter to relation_get must be a boolean')
 
         if is_app:
-            version = JujuVersion.from_environ()
+            version = JujuVersion.from_context(self._juju_context)
```

Comment:
Maybe we should parse and store `_juju_version` once in the constructor?

Reply:
Yes, it makes sense because there are two occurrences of it already. Will do it.